### DISCUSSION TITLE
Is that Confirmation Dialogue really necessary?

### DISCUSSION THREAD

**Discussion by wessleyn** [vkbo/novelWriter#2053] [Ideas]
When I attempt to delete a document or folder, I encounter an unnecessary "Are you sure?" confirmation dialogue. Since files in Nw are not permanently deleted but simply moved, this extra confirmation feels redundant and slows down the workflow.

I believe this confirmation prompt can be safely removed as there’s no real risk of losing the file—it’s just being moved to another directory. This change would improve efficiency by eliminating a step that adds unnecessary friction to the file management process.

**Comment by vkbo** [comment]
novelWriter preserves the order of documents, so an unintended move to trash is still destructive. You can always press keyboard shortcuts by accident, or think you were pressing delete in the editor but were actually in the project tree and thus moved a document without even noticing it. These are why the dialog is there.

If you drag a document to Trash, there is no dialog because that is a very deliberate action.

**Reply by wessleyn** [reply]
Oh .that's a possibility.

Then can you add an option in the settings to remove those dialogues since you wouldn't be surprised if you had the setting off

**Reply by xahodo** [reply]
You could require the destructive keys to be pressed twice in short succession to actually perform the action.

EDIT: Even better, you could only enable this requirement when the dialogs are disabled.
